alacritty_terminal/src/config/bell.rs

```diff
@@ -0,0 +1,106 @@
+use std::time::Duration;
+
+use log::error;
+use serde::{Deserialize, Deserializer};
+use serde_yaml::Value;
+
+use crate::config::{failure_default, Program, LOG_TARGET_CONFIG};
+use crate::term::color::Rgb;
+
+const DEFAULT_BELL_COLOR: Rgb = Rgb { r: 255, g: 255, b: 255 };
+
+#[serde(default)]
+#[derive(Deserialize, Clone, Debug, PartialEq, Eq)]
+pub struct BellConfig {
+    /// Visual bell animation function.
+    #[serde(deserialize_with = "failure_default")]
+    pub animation: BellAnimation,
+
+    /// Visual bell duration in milliseconds.
+    #[serde(deserialize_with = "failure_default")]
+    duration: u16,
+
+    // FIXME RGB deserialization always successedes and uses #000000 color. However it should
+    // fallback to a proper color. So it should be reworked either with
+    // https://github.com/alacritty/alacritty/pull/3507 or after that PR will land, since it'll
+    // allow to impl manual deserialization properly.
```

Comment:
Adding a reminder that this needs to be done before the PR can be merged.

Reply:
I mean it's not a bug from that PR so I'm not sure how it's related and prevents it from being merged, I've just highlighted a very old issue.